cdap-unit-test/src/test/java/io/cdap/cdap/test/app/TestFrameworkTestRun.java

```diff
@@ -1265,169 +1314,169 @@ public Integer call() throws Exception {
       // - if it is in an attempt of a nested transaction, then the expected value is null
       // - if it is in an implicit transaction, then the expected value is "default"
       // - if it is in an explicit transaction, then the expected value is the transaction timeout
-      Object[][] writesToValidate = new Object[][] {
+      Object[][] writesToValidate = new Object[][]{
 
         // transactions attempted by the workers
-        { AppWithCustomTx.WORKER_TX, AppWithCustomTx.INITIALIZE, txDefaulTimeoutWorker },
-        { AppWithCustomTx.WORKER_TX, AppWithCustomTx.INITIALIZE_NEST, AppWithCustomTx.FAILED },
-        { AppWithCustomTx.WORKER_TX, AppWithCustomTx.DESTROY, txDefaulTimeoutWorker },
-        { AppWithCustomTx.WORKER_TX, AppWithCustomTx.DESTROY_NEST, AppWithCustomTx.FAILED },
-        { AppWithCustomTx.WORKER_TX, AppWithCustomTx.RUNTIME, null },
-        { AppWithCustomTx.WORKER_TX, AppWithCustomTx.RUNTIME_TX_D, txDefaulTimeoutWorker},
-        { AppWithCustomTx.WORKER_TX, AppWithCustomTx.RUNTIME_TX, AppWithCustomTx.TIMEOUT_WORKER_RUNTIME },
-        { AppWithCustomTx.WORKER_TX, AppWithCustomTx.RUNTIME_NEST, AppWithCustomTx.FAILED },
-        { AppWithCustomTx.WORKER_NOTX, AppWithCustomTx.INITIALIZE, null },
-        { AppWithCustomTx.WORKER_NOTX, AppWithCustomTx.INITIALIZE_TX, AppWithCustomTx.TIMEOUT_WORKER_INITIALIZE },
-        { AppWithCustomTx.WORKER_NOTX, AppWithCustomTx.INITIALIZE_TX_D, txDefaulTimeoutWorker },
-        { AppWithCustomTx.WORKER_NOTX, AppWithCustomTx.INITIALIZE_NEST, AppWithCustomTx.FAILED },
-        { AppWithCustomTx.WORKER_NOTX, AppWithCustomTx.DESTROY, null },
-        { AppWithCustomTx.WORKER_NOTX, AppWithCustomTx.DESTROY_TX, AppWithCustomTx.TIMEOUT_WORKER_DESTROY },
-        { AppWithCustomTx.WORKER_NOTX, AppWithCustomTx.DESTROY_TX_D, txDefaulTimeoutWorker },
-        { AppWithCustomTx.WORKER_NOTX, AppWithCustomTx.DESTROY_NEST, AppWithCustomTx.FAILED },
-        { AppWithCustomTx.WORKER_NOTX, AppWithCustomTx.RUNTIME, null },
-        { AppWithCustomTx.WORKER_NOTX, AppWithCustomTx.RUNTIME_TX, AppWithCustomTx.TIMEOUT_WORKER_RUNTIME },
-        { AppWithCustomTx.WORKER_NOTX, AppWithCustomTx.RUNTIME_TX_D, txDefaulTimeoutWorker },
-        { AppWithCustomTx.WORKER_NOTX, AppWithCustomTx.RUNTIME_NEST, AppWithCustomTx.FAILED },
+        {AppWithCustomTx.WORKER_TX, AppWithCustomTx.INITIALIZE, txDefaulTimeoutWorker},
```

Comment:
is it just spacing that's changed? If so, please revert in order to restrict the diff to only lines that are relevant to the change.